lib/Zonemaster/Backend/RPCAPI.pm

```diff
@@ -646,11 +646,11 @@ sub jsonrpc_validate {
 
         my @error_response;
 
-        # NOTE: Make that configurable
+        # TODO: Make that configurable
         my $default_language =  'en_US';
         my ($locale, $locale_error) = $self->_get_locale( $jsonrpc_request->{params} );
 
-        # NOTE: temporary workaround to not break everything
+        # remove language param if not present in the real method parameters
```

Comment:
Why is it removed?

Reply:
I first changed the comment to keep the implementation at a global lebel, now that the implementation has moved to only a few specific methods this is not useful anymore.